McCorrections/src/EventWeights_HH.cpp

```diff
@@ -56,17 +56,23 @@ ntuple::ProdSummary EventWeights_HH::GetSummaryWithWeights(const std::shared_ptr
     static const WeightingMode shape_weights(WeightType::PileUp, WeightType::BSM_to_SM, WeightType::DY,
                                              WeightType::TTbar, WeightType::Wjets, WeightType::GenEventWeight);
     static const WeightingMode shape_weights_withTopPt = shape_weights | WeightingMode(WeightType::TopPt);
+    static const WeightingMode shape_weights_withPileUp = shape_weights | WeightingMode(WeightType::PileUp);
 
     auto summary_tuple = ntuple::CreateSummaryTuple("summary", file.get(), true, ntuple::TreeState::Full);
     auto summary = ntuple::MergeSummaryTuple(*summary_tuple);
     summary.totalShapeWeight = 0;
     summary.totalShapeWeight_withTopPt = 0;
```

Comment:
totalShapeWeight_withPileUp_Up and totalShapeWeight_withPileUp_Down should be set to 0